v3/viktus/sp111/sp111_v2.json

```diff
@@ -0,0 +1,526 @@
+{
+  "name": "SP-111",
+  "vendorId": "0x5644",
+  "productId": "0x5112",
+  "keycodes": [ "qmk_lighting" ],
+  "matrix": {
+    "rows": 12,
+    "cols": 11
+  },
+  "menus": [
+    {
+      "label": "Lighting",
+      "content": [
+        {
+          "label": "Caps Lock Indicator",
+          "content": [
+            {
+              "label": "Brightness",
+              "type": "range",
+              "options": [0, 255],
+              "content": ["id_caps_indicator_brightness", 1, 2]
+            },
+            {
+              "label": "Color",
+              "type": "color",
+              "content": ["id_caps_indicator_color", 1, 3]
+            }
+          ]
+        },
+        {
+          "label": "Scroll Lock Indicator",
+          "content": [
+            {
+              "label": "Brightness",
+              "type": "range",
+              "options": [0, 255],
+              "content": ["id_scroll_indicator_brightness", 1, 5]
+            },
+            {
+              "label": "Color",
+              "type": "color",
+              "content": ["id_scroll_indicator_color", 1, 6]
+            }
+          ]
+        },
+        {
+          "label": "Num Lock Indicator",
+          "content": [
+            {
+              "label": "Brightness",
+              "type": "range",
+              "options": [0, 255],
+              "content": ["id_caps_indicator_brightness", 1, 8]
+            },
+            {
+              "label": "Color",
+              "type": "color",
+              "content": ["id_caps_indicator_color", 1, 9]
+            }
+          ]
+        }
+      ]
+    }
+  ],
```

Comment:
Unfortunately we require this to also be in QMK source code. So you will have to remove this.
This is so that the automatic builds on the website align with the VIA source.

It is unfortunate that QMK doesn't accept this code but it is understandable as it would be copy and pasted. There is some effort to move indicators into core, but there is not ETA at this time. :(